SourceX/storm/storm.cpp

```diff
@@ -604,12 +636,20 @@ BOOL SVidPlayContinue(void)
 		int scaledH = SVidHeight * factor;
 
 		SDL_Rect pal_surface_offset = { (SCREEN_WIDTH - scaledW) / 2, (SCREEN_HEIGHT - scaledH) / 2, scaledW, scaledH };
+#ifdef USE_SDL1
+		SDL_Surface *tmp = SDL_ConvertSurface(SVidSurface, window->format, 0);
+		if (SDL_BlitSurface(tmp, NULL, surface, &pal_surface_offset) <= -1) {

```

Comment:
This should have a TODO comment as a minimum

Reply:
The issue here is that SDL2 uses an int for pixel format, while SDL1 uses a  complex struct with fields and pointers to other structs.

I don't think we want to convert the struct to int and back in the backport layer (that'd be complex and error prone, as the internals of SDL2 are quite different from SDL1).

Reply:
Oh I see that you probably meant that the issue is that it doesn't scale. I've now added a todo.